### PR TITLE
Mpc outer loop

### PR DESCRIPTION
**Pull request by isacbo1**
Changed trajectory controller from LQR to MPC in Main_bikesim.slx, added MPC parameters to main_sim.m. Old LQR bike model moved to Main_bikesim_old.slx.
Added MPC_tuning.m which runs batch simulations for tuning the MPC controller, and added plotting tool which compares it to the LQR controller.
Fixed sharp turn and smooth curve reference generation.

### CODE REVIEW

**Comment by Laghund** [comment]
Update the documentation with your changes, if applicable

**Comment by isacbo1** [comment]
From what I have found, the documentation is only about working on the physical bike and setting up labview etc. I have not found any documentation on the simulation part, which this would be part of. So feels weird to only add docs about the MPC simulation tuning, unless we are to write documentation for the rest of the simulation files

File: simulation/Plot_MPC_tuning_results.m [hunk 1]
```diff
@@ -0,0 +1,355 @@
+function [] = Plot_MPC_tuning_results(Ts, foldername)
+% Trajectory
+load("batch_simulations\" + foldername + "\tuning_setup.mat", "sim_tbl")
```
Comment (Laghund) [inline simulation/Plot_MPC_tuning_results.m:3]: Path backslashes don't work on unix but I think forward slashes work on windows. I.e. `load("batch_simulations/" + foldername + "/tuning_setup.mat", "sim_tbl")` should work on both.

Comment (Laghund) [reply simulation/Plot_MPC_tuning_results.m:3]: (There are more instances than this one)

**Comment by Laghund** [comment]
Great job!  
Imo some short documentation on how to use the simulations would still be nice. Suggestions:  
* Change `docs/introduction.md` line 47: "A Simulink model of the bike exists, but is not present in this repository."  
* To `docs/manual.md` add a short "Simulation" section that at least tells you which file to run as the entrypoint (`simulation/Main_sim.m`?), and which Simulink file(s) is/are then used.

**Pull request merged**